bigtable/beam/helloworld/pom.xml

```diff
@@ -37,7 +37,10 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <apache_beam.version>2.31.0</apache_beam.version>
+    <!-- Keep this apache_beam.version in sync with version used in bigtable-hbase-beam -->
```

Comment:
Let's add an exclusion to the[ renovate config](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/renovate.json) so this doesn't accidentally get updated in the future.